src/mongocrypt-kms-ctx.c

```diff
@@ -1517,6 +1601,9 @@ _mongocrypt_kms_ctx_init_kmip_get (mongocrypt_kms_ctx_t *kms_ctx,
                                    const char *unique_identifier,
                                    _mongocrypt_log_t *log)
 {
+   BSON_ASSERT_PARAM (kms_ctx);
+   BSON_ASSERT_PARAM (endpoint);
```

Comment:
```suggestion
   BSON_ASSERT_PARAM (kms_ctx);
   BSON_ASSERT_PARAM (endpoint);
   BSON_ASSERT_PARAM (unique_identifier);
```
Assert `unique_identifier` is not-null.